vocabularies/Org.OData.Core.V1.xml

```diff
@@ -510,9 +510,13 @@ Any simple identifier | Any type listed in `Validation.OpenPropertyTypeConstrain
       </Term>
       <ComplexType Name="OptionalParameterType">
         <Property Name="DefaultValue" Type="Edm.String" Nullable="true">
-          <Annotation Term="Core.Description" String="Default value for an optional parameter of primitive or enumeration type, using the same rules as the `cast` function in URLs." />
+          <Annotation Term="Core.Description" String="Default value for an optional parameter of primitive type, using the same rules as the `cast` function in URLs" />
           <Annotation Term="Core.LongDescription" String="If no explicit DefaultValue is specified, the service is free on how to interpret omitting the parameter from the request. For example, a service might interpret an omitted optional parameter `KeyDate` as having the current date." />
         </Property>
+        <Property Name="NonPrimitiveDefaultValue" Type="Edm.Untyped" Nullable="true">

```

Comment:
The vocabulary currently uses OData 4.0, which doesn't have Edm.Untyped:
https://github.com/oasis-tcs/odata-vocabularies/blob/59e19bb1eaf27e88037aacf9a04a2b8562c9997e/vocabularies/Org.OData.Core.V1.xml#L41

Do we have to create a new vocabulary `Org.OData.Core.V2`?

Reply:
Or use `Type=Collection(Edm.String)` which allows at least specifiying an empty collection as default.

Reply:
> Do we have to create a new vocabulary `Org.OData.Core.V2`?

That's too heavy a tool for this small problem.